src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
+    .split(';')
+    .reduce((objWithCSS, style) => {
+      const [property, value] = style.trim().split(':');
+
+      if (value) {
+        return {
+          ...objWithCSS,
+          [property.trim()]: value.trim(),
+        };
+      }
+
+      return {
+        ...objWithCSS,
+      };
```

Comment:
1. You could write it shorter
2. I think it is better to use another name for variable (not obj etc...)
```suggestion
      return objWithCSS;
```